### PR TITLE
Fragment work

### PR DESCRIPTION
**Pull request by lramaswamy**
The following workflow has been completed:
1) the user logs in: Sees the travel prep fragment
2) If the user click on proceed to checkin, then the scan boarding fragment is shown
3) Once the user scans (pretend scan now, needs to be implemented)..click on the bar code image -> Summary page is shown
4) In the summary page, if the user hits on start travel...the card layout is shown. (Card layout is still TBD...it should show all the stages past checkin)

### CODE REVIEW

File: app/src/main/java/com/exotikosteam/exotikos/fragments/TravelStatusFragment.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        super.onViewCreated(view, savedInstanceState);
+        populateViewObjects(view);
+    }
+
+    private void populateViewObjects(View view) {
+
+        //@TODO need a solid logic for this...for now I am putting in mock data in static views...
+        //Assumption: Flight 0 will be the first top flight...any further flights will
+        // be additional stops and the last entry will be the destination
+        // For now, I am assuming 1 departure, 1 stop and 1 destination
```
Comment (adajazbor) [inline app/src/main/java/com/exotikosteam/exotikos/fragments/TravelStatusFragment.java:82]: In FlightStatus you have a two fields: currentFlight, and flightStep. They shows you what is the current status of whole trip. Also Flight has filed order - to keep flights in proper order

File: app/src/main/java/com/exotikosteam/exotikos/fragments/TravelStatusFragment.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    @BindView(R.id.tvTravelDate1) TextView tvTravelDate1;
+    @BindView(R.id.tvTravelDest1) TextView tvTravelDest1;
+    @BindView(R.id.tvDepartureTime1) TextView tvDepartureTime1;
+    @BindView(R.id.tvStop1) TextView tvStop1;
+    @BindView(R.id.tvArrivalTime1) TextView tvArrivalTime1;
+    @BindView(R.id.tvDepartureTime2) TextView tvDepartureTime2;
+    @BindView(R.id.tvDestination) TextView tvDestination;
+    @BindView(R.id.tvArrivalTime2) TextView tvArrivalTime2;
+
+
```
Comment (adajazbor) [inline app/src/main/java/com/exotikosteam/exotikos/fragments/TravelStatusFragment.java:46]: Why you do not use databinding library?

File: app/src/main/java/com/exotikosteam/exotikos/fragments/TravelStatusFragment.java (hunk 1 above)
Comment (lramaswamy) [reply app/src/main/java/com/exotikosteam/exotikos/fragments/TravelStatusFragment.java:82]: yes. I have integrated that. This was older comment.

File: app/src/main/java/com/exotikosteam/exotikos/fragments/TravelStatusFragment.java (hunk 2 above)
Comment (lramaswamy) [reply app/src/main/java/com/exotikosteam/exotikos/fragments/TravelStatusFragment.java:46]: Will make those changes now...used ButterKnife instead for now...

**Pull request merged**